### PR TITLE
Updates to docs

### PR DESCRIPTION
**Pull request by duckranger**
Added smtp
Updated build connector tutorial

### CODE REVIEW

File: docs/building-connectors.md [hunk 1]
```diff
@@ -1,114 +1,136 @@
...
-class MyConnector{
+class MyConnector {
     
-constructor(connectorOptions, id) {...}
+  constructor(options, id) {...}
+
+  onStart(app) {...}
 
-start(app) {...}
+  onStop() {...}
```
Comment (ashevat) [inline docs/building-connectors.md:23]: I think need to have both the stop and start and the onstop and on start. You need to implement the former if you are not inheriting from base connector.  Especially in light of the paragraph above.

File: docs/connectors.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Reshuffle achieves this by using **Connectors** as the connection mechanism to a
 Reshuffle ships with an existing ecosystem comprising several connectors you can use to connect to common systems. A list of these Reshuffle-provided connectors can be found in this page, and their code can be found in the various repositories [here](https://github.com/reshufflehq/).  
 
 The framework is extendable, meaning you can create connectors to services you use or even your own proprietary systems. 
-The process for building new connectors is described in the [manual](./building-connectors.md). 
+The process for building new connectors is described in the [manual]("./buidling-connectors.md").
```
Comment (ashevat) [inline docs/connectors.md:9]: The link here is broken, the removed link works and tested.

File: docs/building-connectors.md (hunk 1 above)
Comment (duckranger) [reply docs/building-connectors.md:23]: Good point. If you don't use `BaseConnector` then you need start and stop and if you do use it you need `onStart` and `onStop`. What would be a clear way to explain this?

File: docs/connectors.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Reshuffle achieves this by using **Connectors** as the connection mechanism to a
 Reshuffle ships with an existing ecosystem comprising several connectors you can use to connect to common systems. A list of these Reshuffle-provided connectors can be found in this page, and their code can be found in the various repositories [here](https://github.com/reshufflehq/).  
 
 The framework is extendable, meaning you can create connectors to services you use or even your own proprietary systems. 
-The process for building new connectors is described in the [manual](./building-connectors.md). 
+The process for building new connectors is described in the [manual](./buidling-connectors.md).
```
Comment (ashevat) [inline docs/connectors.md:9]: this link it still broken. seems like a merge issue. it should be [manual](./building-connectors.md)

**Review by ashevat — CHANGES_REQUESTED** [review]

**Review by ashevat — APPROVED** [review]

**Pull request merged**